keycloak-rest-api-extensions/src/main/java/io/cloudtrust/keycloak/credential/Ssha1PasswordHashProviderFactory.java

```diff
@@ -0,0 +1,39 @@
+package io.cloudtrust.keycloak.credential;
+
+import org.keycloak.Config;
+import org.keycloak.credential.hash.PasswordHashProvider;
+import org.keycloak.credential.hash.PasswordHashProviderFactory;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.models.KeycloakSessionFactory;
+
+/**
+ * Provider factory for the Salted SHA1 password hash algorithm.
+ */
+public class Ssha1PasswordHashProviderFactory implements PasswordHashProviderFactory {
```

Comment:
Better than a copy/paste, we could have an AbstractPasswordHashProviderFactory and extends them as Ssha1PasswordHashProviderFactory and Ssha256PasswordHashProviderFactory